apps/taxes/src/modules/taxjar/calculate-taxes/taxjar-calculate-taxes-payload-transformer.ts

```diff
@@ -9,19 +9,19 @@ export class TaxJarCalculateTaxesPayloadTransformer {
   constructor(private readonly config: TaxJarConfig) {}
 
   transform(taxBase: TaxBaseFragment, matches: TaxJarTaxCodeMatches): TaxJarCalculateTaxesTarget {
-    const fromAddress = taxJarAddressFactory.fromChannelToTax(this.config.address);
+    const from = taxJarAddressFactory.fromChannelToTax(this.config.address);
```

Comment:
question: why changing this name? I think the previous one was actually more readable 😅